treeherder/log_parser/artifactbuilders.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-from .parsers import PerformanceParser, StepParser, TinderboxPrintParser
```

Comment:
I think it's beyond the scope of this PR, but the `StepParser` has got to go.  :)  I has a `sub_parser` called the `ErrorParser` that is still valuable.  I *think* you could just promote the `ErrorParser` up to where the `StepParser` is now and it might work.  Worth trying...  I created: https://bugzilla.mozilla.org/show_bug.cgi?id=1633041 for this.

Reply:
Yeah, I can definitely do a follow up pr to clean up some more of this while it's still fresh in my mind.

Reply:
The more tedious aspect is cleaning up old tests/updating the data but needs to be done also.